presto-plugin-toolkit/src/main/java/com/facebook/presto/plugin/base/security/ForwardingConnectorAccessControl.java

```diff
@@ -205,4 +205,22 @@ public void checkCanSetRole(ConnectorTransactionHandle transactionHandle, Connec
     {
         delegate().checkCanSetRole(transactionHandle, identity, role, catalogName);
     }
+
+    @Override
+    public void checkCanShowRoles(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, String catalogName)
```

Comment:
https://github.com/prestosql/presto/pull/90/commits/4f4b268adce8a214b73344a82c872bd493ef55ac#diff-cc0689ca452d7c2209c27e411d80c45aR177 also adds `checkCanSetRole`

Reply:
Actually that is mistake. This method has to be moved to the `Add access control checks for SHOW ROLES`